hibernate-ucp/hibernate-ucp.gradle

```diff
@@ -0,0 +1,18 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+description = 'Integration for Oracle UCP into Hibernate O/RM'
+
+apply from: rootProject.file( 'gradle/published-java-module.gradle' )
+
+dependencies {
+    implementation project( ':hibernate-core' )
+    implementation libs.ucp
+    implementation libs.ojdbc11
```

Comment:
Is it on purpose that UCP requires the ojdbc11 driver? Also, I'd prefer if we reused the `dbLibs.oracle` dependency rather than introducing a new one.

Reply:
ucp11.jar has a dependency on ojdbc11.jar and both must come from the same release.
An alternative approach would be to add ucp11.jar to dbLibs along with ojdbc11.jar.